classes/controller/AdminController.php

```diff
@@ -2622,7 +2626,10 @@ public function setMedia()
         ));
 
         //loads specific javascripts for the admin theme
-        $this->addJS(__PS_BASE_URI__.$this->admin_webpath.'/themes/'.$this->bo_theme.'/js/vendor/bootstrap.min.js');
+        $this->addJS(__PS_BASE_URI__.$this->admin_webpath.'/themes/'.$this->bo_theme.'/node_modules/PrestaKit/dist/js/tether.min.js');
```

Comment:
you're using here entire path dozen of times, maybe it is good to create one method for this?